service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/TypeOfDocument.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+@SuppressWarnings("membername")
+public class TypeOfDocument {
```

Comment:
Please rename, it is not really type of document, it is more reference to document in document store.

Reply:
Named to DocumentReference, what do you think? @geek-soft